src/pages/timecard/Timecard.jsx

```diff
@@ -113,12 +119,25 @@ const Timecard = () => {
         </Link>
       </div>
 
-      {timeEntries.map((timeEntry, index) => (
-        <div key={index} className="govuk-!-margin-bottom-6">
-          <EditShiftTimecard timeEntry={timeEntry} timeEntriesIndex={index} />
-        </div>
-      ))}
-      {timeEntries.length === 0 && <SelectTimecardPeriodType />}
+      {newTimeEntry && <SelectTimecardPeriodType />}
+      {!newTimeEntry && (
+        <>
+          {timeEntries.map((timeEntry, index) => (
+            <div key={index} className="govuk-!-margin-bottom-6">
+              <EditShiftTimecard
+                timeEntry={timeEntry}
+                timeEntriesIndex={index}
+              />
+            </div>
+          ))}
+          {timeEntries.length === 0 && (
+            <AddTimeCardPeriod timecardEmpty={true} />
+          )}
+          {timeEntries.length > 0 && (
+            <AddTimeCardPeriod timecardEmpty={false} />
+          )}
```

Comment:
```suggestion
          <AddTimeCardPeriod timecardEmpty={timeEntries.length === 0} />
```
Could this be done in one line instead?